### PR TITLE
refactor: generate quota APIs by go-swagger

### PR DESCRIPTION
**Pull request by heww**
Signed-off-by: He Weiwei <hweiwei@vmware.com>

### CODE REVIEW

File: src/server/v2.0/handler/quota.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	}
+
+	query := &q.Query{
+		Keywords: q.KeyWords{
+			"reference":    lib.StringValue(params.Reference),
+			"reference_id": lib.StringValue(params.ReferenceID),
+		},
+		PageNumber: *params.Page,
+		PageSize:   *params.PageSize,
+		Sorting:    lib.StringValue(params.Sort),
```
Comment (wy65701436) [inline src/server/v2.0/handler/quota.go:67]: the sorting could be empty here, what about if it's a empty tring?

File: src/server/v2.0/handler/model/quota.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+	hard, err := q.GetHard()
+	if err != nil {
+		fields := log.Fields{"quota_id": q.ID, "error": err}
+		log.G(ctx).WithFields(fields).Warningf("failed to get hard from quota")
+	}
+
+	used, err := q.GetUsed()
+	if err != nil {
+		fields := log.Fields{"quota_id": q.ID, "error": err}
```
Comment (wy65701436) [inline src/server/v2.0/handler/model/quota.go:45]: What's the value of hard or used on error happen? What's the vaule in the response?

**Review by wy65701436 — APPROVED** [review]
lgtm

File: src/server/v2.0/handler/quota.go (hunk 1 above)
Comment (heww) [reply src/server/v2.0/handler/quota.go:67]: When `Sorting` is an empty string, the DAO of the quota will list by the creation_time in descending order.

File: src/server/v2.0/handler/model/quota.go (hunk 2 above)
Comment (heww) [reply src/server/v2.0/handler/model/quota.go:48]: Now, it's changed to return empty map when getting hard or used attribute failed.

**Review by ywk253100 — APPROVED** [review]

**Pull request merged**